homeassistant/components/adguard/sensor.py

```diff
@@ -178,7 +178,7 @@ def __init__(self, adguard):
         """Initialize AdGuard Home sensor."""
         super().__init__(
             adguard,
-            "Searches Safe Search Enforced",
+            "AdGuard Safe Search Enforced",
```

Comment:
Let us make it proper English in that case:
```suggestion
            "AdGuard Safe Searches Enforced",
```